phys2bids/interfaces/txt.py

```diff
@@ -9,34 +9,27 @@
 from phys2bids.physio_obj import BlueprintInput
 
 
-def populate_phys_input(filename, chtrig):
+def labchart_read(channel_list, chtrig, header=[]):
     """
-    Populate object phys_input
-    for now this works only with labchart files
-        Input (Properties)
-    ------------------
-    filename: str
-        path to the txt labchart file
+    Reading function for labchart files
+    Parameters
+    ----------
+    channel_list: list
+        list with channels only
     chtrig : int
         index of trigger channel
-
-    Output
-    ------------------
-    BlueprintInput object for more see BlueprintInput docs
+    header: list
+        list with that contains file header
+    Returns
+    -------
+    BlueprintInput
+    See Also
+    --------
+    physio_obj.BlueprintInput
     """
-    header = []
-    channel_list = []
-    with open(filename, 'r') as f:
-        for line in f:
-            line = line.rstrip('\n').split('\t')
-            try:
-                float(line[0])
-            except ValueError:
-                header.append(line)
-                continue
-            line = [float(i) for i in line]
-            channel_list.append(line)
     # get frequency
+    if len(header) == 0:
+        raise AttributeError('Files without header are not supported yet')
     interval = header[0][1].split(" ")
     if interval[-1] not in ['hr', 'min', 's', 'ms', 'µs']:
         raise AttributeError(f'Interval unit "{interval[-1]}" is not in a valid LabChart'
```

Comment:
Can you add a comment with the reference of the valid LabChart units (if it exists)?

Reply:
Added in the docstring